packages/app/src/App.tsx

```diff
@@ -1,26 +1,27 @@
 import { FC } from "react";
 import { Route, Switch } from "react-router";
 import { BrowserRouter as Router } from "react-router-dom";
+import { Grid } from "grommet";
 
 import TestnetBanner from "./components/testnet-banner";
 import Home from "./pages/home";
 import Token from "./pages/token";
 import { NotificationsList } from "./components/transactions";
-import Foot from './components/footer'
-import {Grid, Box} from 'grommet'
+import Foot from "./components/footer";
+
 const App: FC = () => {
   return (
     <>
-      <Grid fill rows={["xxsmall", "flex", "xxsmall"]}>
+      <Grid rows={["xxsmall", "flex", "xxsmall"]}>
         <TestnetBanner />
         <Router>
           <Switch>
             <Route exact path="/" component={Home} />
             <Route path="/stakers/:id" component={Token} />
           </Switch>
         </Router>
-        <Foot />
       </Grid>
+      <Foot />
```

Comment:
This moves the footer out of the grid which assigns xxsmall to the row for it

Reply:
I fixed the grid layout in a good way now on this branch, but the branch you have merged changed a whole lot of things :)